architecture-decision-records/0030-metadata-input-validation.md

```diff
@@ -0,0 +1,28 @@
+# 4. Metadata input validation
```

Comment:
Number should match the ADR number, ie: `... 30. Metadata input ...`

Reply:
Good catch, I have updated it, thanks.